app/src/main/res/layout/pin_password_activity.xml

```diff
@@ -1,113 +1,134 @@
-      android:textSize="24sp"
-      android:layout_marginTop="12dp"
-      android:paddingBottom="12dp"
-      app:layout_constraintTop_toBottomOf="@+id/hello_text"
+      android:textColor="@color/oppiaPrimaryText"
+      android:textSize="18sp"
+      app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintEnd_toEndOf="parent"/>
+      app:layout_constraintTop_toBottomOf="@+id/hello_text_view" />
+
     <TextView
-      android:id="@+id/error_text"
+      android:id="@+id/pin_password_error_text_view"
       android:layout_width="wrap_content"
-      android:layout_height="24dp"
+      android:layout_height="wrap_content"
       android:text="@{viewModel.showError ? @string/pin_password_incorrect_pin : null}"
-      android:textColor="@color/red"
-      android:textSize="20sp"
-      app:layout_constraintTop_toBottomOf="@+id/enter_text"
+      android:textColor="@color/oppiaBrown"
+      android:textSize="16sp"
+      android:layout_marginTop="24dp"
+      app:layout_constraintEnd_toEndOf="parent"
       app:layout_constraintStart_toStartOf="parent"
-      app:layout_constraintEnd_toEndOf="parent"/>
+      app:layout_constraintTop_toBottomOf="@+id/enter_text_view" />
+
     <com.chaos.view.PinView
       android:id="@+id/input_pin"
       style="@style/PinWidget.PinView"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
       android:cursorVisible="true"
       android:inputType="@{viewModel.showPassword ? InputType.TYPE_CLASS_NUMBER : (InputType.TYPE_CLASS_NUMBER | InputType.TYPE_NUMBER_VARIATION_PASSWORD)}"
-      android:padding="10dp"
       android:itemBackground="@color/white"
-      android:textColor="@color/black"
-      android:textSize="22sp"
-      android:textStyle="bold"
+      android:textColor="@color/oppiaPrimaryText"
+      android:textSize="20sp"
+      android:fontFamily="sans-serif"
       android:layout_marginTop="10dp"
-      app:cursorColor="@color/black"
+      app:cursorColor="@color/oppiaPrimaryText"
       app:cursorWidth="2dp"
       app:hideLineWhenFilled="false"
       app:itemCount="@{viewModel.profile.pin.length()}"
       app:itemHeight="48dp"
-      app:itemRadius="4dp"
+      app:itemRadius="2dp"
       app:itemSpacing="8dp"
       app:itemWidth="28dp"
-      app:lineColor="@color/grey"
+      app:lineColor="@color/avatarBorder"
       app:lineWidth="1dp"
       app:viewType="rectangle"
-      app:layout_constraintTop_toBottomOf="@+id/error_text"
+      app:layout_constraintTop_toBottomOf="@+id/pin_password_error_text_view"
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintEnd_toEndOf="parent"/>
+
     <TextView
       android:id="@+id/forgot_pin"
       android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:layout_marginTop="10dp"
+      android:layout_height="48dp"
+      android:layout_marginTop="12dp"
```

Comment:
Change this to `android:paddingTop="12dp"` for better clickable item

Reply:
updated.